vm-setup/roles/v1aX_integration_test/vars/main.yml

```diff
@@ -31,10 +31,11 @@ BAREMETALV6_POOL_RANGE_END: "{{ lookup('env', 'BAREMETALV6_POOL_RANGE_END') | de
 EXTERNAL_SUBNET_V6_PREFIX: "{{ lookup('env', 'EXTERNAL_SUBNET_V6_PREFIX') | default('64', true) }}"
 EXTERNAL_SUBNET_V6_HOST: "{{ lookup('env', 'EXTERNAL_SUBNET_V6_HOST') | default('fd55::1', true) }}"
 IRONIC_HOST_IP: "{{ lookup('env', 'IRONIC_HOST_IP') | default('172.22.0.2', true) }}"
-DEPLOY_KERNEL_URL: "{{ lookup('env', 'DEPLOY_KERNEL_URL') }}"
-DEPLOY_RAMDISK_URL: "{{ lookup('env', 'DEPLOY_RAMDISK_URL') }}"
-IRONIC_URL: "{{ lookup('env', 'IRONIC_URL') }}"
-IRONIC_INSPECTOR_URL: "{{ lookup('env', 'IRONIC_INSPECTOR_URL') }}"
+CLUSTER_URL_HOST: "{{ lookup('env', 'CLUSTER_URL_HOST') }}"
+DEPLOY_KERNEL_URL: "http://{{ CLUSTER_URL_HOST }}:6180/images/ironic-python-agent.kernel"
+DEPLOY_RAMDISK_URL: "http://{{ CLUSTER_URL_HOST }}:6180/images/ironic-python-agent.initramfs"
+IRONIC_URL: "http://{{ CLUSTER_URL_HOST }}:6385/v1/"
+IRONIC_INSPECTOR_URL: "http://{{ CLUSTER_URL_HOST }}:5050/v1/"
```

Comment:
you should not override it in the ansible playbook, they should be set right from the start.

Reply:
you are right, I realized that after I did a local run

Reply:
Removed